src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -542,7 +541,6 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
-                        .AddPropertyColumn("HandleCount")
```

Comment:
The `HandleCount` property is also referenced by the view definitions `priority` and `starttime` in the same file, and also `Handles` and `HandleCount` are referenced by the format definition of `ViewsOf_System_Diagnostics_Process_IncludeUserName`. I think they should be updated too.